### PR TITLE
Feat: AUTH

### PR DESCRIPTION
**Pull request by Logan1x**
This PR includes changes below mentioned
- Login, Logout functionality
- auth Context
- installing Axios
- adding user to db

### CODE REVIEW

File: src/backend/controllers/AuthController.js [hunk 1]
```diff
@@ -61,6 +61,7 @@ export const signupHandler = function (schema, request) {
 
 export const loginHandler = function (schema, request) {
   const { email, password } = JSON.parse(request.requestBody);
+  console.log(email, password);
```
Comment (shobit1337) [inline src/backend/controllers/AuthController.js]: Always remove unnecessary comments before pushing your code to Github.

File: src/context/authContext.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import { useNavigate } from "react-router-dom";
+
+const AuthContext = createContext();
+
+const AuthProvider = ({ children }) => {
+  const navigate = useNavigate();
+
+  const localStorageToken = JSON.parse(localStorage.getItem("loginItems"));
+  const [token, setToken] = useState(localStorageToken?.token);
+  const [currUser, setCurrUser] = useState(localStorageToken?.user);
```
Comment (shobit1337) [inline src/context/authContext.js]: currUser is not that readable, I suggest using a more readable name here. `currentUser`

File: src/context/authContext.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+  const loginHandler = async (email, password) => {
+    try {
+      const response = await loginUtility(email, password);
+
+      console.log(response);
```
Comment (shobit1337) [inline src/context/authContext.js]: remove console.log

**Review by shobit1337 — APPROVED** [review]
Suggested some changes; the rest of your code looks clean and well structured. Good work @Logan1x

**Comment by Logan1x** [comment]
image

CC: @shobit1337

**Review by thegeorgenikhil — APPROVED** [review]
The code and UI looks good.
A loader/toast would be better during login, I hope you are planning to implement that later on.
Good to merge!

**Comment by Logan1x** [comment]
Yes I am, Thanks @thegeorgenikhil and @shobit1337

**Pull request merged**